EddiInaraService/InaraService.cs

```diff
@@ -0,0 +1,282 @@
+                            string commanderFrontierID = inaraCredentials?.commanderFrontierID;
+                            DateTime lastSync = inaraCredentials.lastSync;
+                            if (!string.IsNullOrEmpty(inaraCredentials.apiKey) && !string.IsNullOrEmpty(commanderName))
+                            {
+                                instance = new InaraService(lastSync, inaraCredentials.apiKey ?? readonlyAPIkey, commanderName, commanderFrontierID);
+                                Logging.Info("Configuring EDDI access to Inara profile data");
+                            }
+                            else
+                            {
+                                instance = new InaraService(lastSync, readonlyAPIkey);
+                                if (string.IsNullOrEmpty(inaraCredentials.apiKey))
+                                {
+                                    Logging.Warn("Configuring Inara service for limited access: API key not set.");
+                                }
+                                if (string.IsNullOrEmpty(commanderName))
+                                {
+                                    Logging.Warn("Configuring Inara service for limited access: Commander name not detected.");
+                                }
+                            }
+                        }
+                    }
+                }
+                return instance;
+            }
+        }
+
+        public static void Reload()
+        {
+            Logging.Debug("Creating new Inara service instance.");
+            instance = null;
+            _ = Instance;
+        }
+
+        // If you need to do some testing on Inara's API, please set the `inBeta` boolean header property to true.
+        public List<InaraResponse> SendEventBatch(ref List<InaraAPIEvent> events, bool inBeta = false)
+        {
+            List<InaraResponse> inaraResponses = new List<InaraResponse>();
+
+            if (string.IsNullOrEmpty(apiKey))
+            {
+                return inaraResponses;
+            }
+
+            // Flag each event with a unique ID we can use when processing responses
+            List<InaraAPIEvent> indexedEvents = new List<InaraAPIEvent>();
+            for (int i = 0; i < events.Count; i++)
+            {
+                InaraAPIEvent indexedEvent = events[i];
+                indexedEvent.eventCustomID = i;
+                indexedEvents.Add(indexedEvent);
+            }
+
+            var client = new RestClient("https://inara.cz/inapi/v1/");
+            var request = new RestRequest(Method.POST);
+            InaraSendJson inaraRequest = new InaraSendJson()
+            {
+                header = new Dictionary<string, object>()
+                {
+                    { "appName", "EDDI" },
+                    { "appVersion", Constants.EDDI_VERSION.ToString() },
+                    { "isDeveloped", !inBeta },
```

Comment:
My reading if the Inara API docs is that we use inBeta like this for a beta build of EDDI running with a live build of the game, whereas for a beta build of the game we send nothing at all. It's a bit hard to check this from the web view so I'll check it in VS.

Reply:
You might be right. The Inara docs state:
`"Do NOT send data from the game beta versions (handle it in your applications). But you can use these data on your test accounts, of course."` 
I was thinking that the second clause covered the approach I used but if not then we'd essentially want to disable to Inara responder whenever the `Beta` property in `EliteConfiguration` is true.